go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/aws/eks-hybrid
 
-go 1.22
+go 1.22.0
```

Comment:
i suspect u can revret all of this. this happens usually because the cdk node_modules folder exists and go get/mod for some reason sees stuff in there and adds it.